### PR TITLE
[REF] web: move crash manager to frontend only

### PR DESCRIPTION
**Pull request by sdegueldre**

### CODE REVIEW

File: addons/web/static/src/legacy/js/_deprecated/crash_manager_warning_dialog.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/** @odoo-module default=false */
```
Comment (aab-odoo) [inline addons/web/static/src/legacy/js/_deprecated/crash_manager_warning_dialog.js]: default=false? why?

File: addons/web/static/src/legacy/js/core/misc.js [hunk 2]
```diff
@@ -101,9 +100,6 @@ function unblockUI() {
  * If wait is true, sleep 1s and wait for the server i.e. after a restart.
  */
 function redirect (url, wait) {
-    // Dont display a dialog if some xmlhttprequest are in progress
-    disableCrashManager();
```
Comment (aab-odoo) [inline addons/web/static/src/legacy/js/core/misc.js:105]: not necessary anymore?

File: addons/web/static/src/legacy/js/fields/basic_fields.js [hunk 3]
```diff
@@ -1898,6 +1898,7 @@ var AbstractFieldBinary = AbstractField.extend({
     on_file_uploaded: function (size, name) {
         if (size === false) {
             this.displayNotification({ message: _t("There was a problem while uploading your file"), type: 'danger' });
+            // TODO: adapt comment
```
Comment (aab-odoo) [inline addons/web/static/src/legacy/js/fields/basic_fields.js]: joli

Comment (aab-odoo) [reply addons/web/static/src/legacy/js/fields/basic_fields.js]: not sure we want to use the crashmanager for this anyway

File: addons/web/static/src/legacy/js/owl_compatibility.js [hunk 4]
```diff
@@ -245,7 +245,7 @@ odoo.define('web.OwlCompatibility', function () {
                     payload.callback(result);
                 } else {
                     throw new Error(
-                        `The service "${service}" is not present in the legacy owl environment.
+                        `The service "${payload.service}" is not present in the legacy owl environment.
```
Comment (aab-odoo) [inline addons/web/static/src/legacy/js/owl_compatibility.js:248]: lol

File: addons/web/static/src/legacy/js/public/public_crash_manager.js [hunk 5]
```diff
@@ -1,31 +1,398 @@
...
+            }
+            this._super();
+        }
+    });
+
+    var ErrorDialog = CrashManagerDialog.extend({
+        template: 'CrashManager.error',
+    });
+
+    var WarningDialog = CrashManagerDialog.extend({
```
Comment (aab-odoo) [inline addons/web/static/src/legacy/js/public/public_crash_manager.js]: 2 implementations of this dialog?

File: addons/website_forum/static/src/js/website_forum.js [hunk 6]
```diff
@@ -248,7 +248,7 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
         }
         this.call('crash_manager', 'show_warning', {
             message: msg,
-            title: title,
+            type: title,
```
Comment (aab-odoo) [inline addons/website_forum/static/src/js/website_forum.js]: can we avoid this?

File: addons/web/static/src/legacy/js/owl_compatibility.js (hunk 4 above)
Comment (sdegueldre) [reply addons/web/static/src/legacy/js/owl_compatibility.js:248]: todo: backport in stable

Comment (sdegueldre) [reply addons/web/static/src/legacy/js/owl_compatibility.js:248]: Todo extract in separate commit to !fixup on main commit (rewrite webclient in owl)

File: addons/website_forum/static/src/js/website_forum.js (hunk 6 above)
Comment (sdegueldre) [reply addons/website_forum/static/src/js/website_forum.js]: remove from PR and do in stable

File: addons/web/static/src/legacy/js/core/misc.js (hunk 2 above)
Comment (sdegueldre) [reply addons/web/static/src/legacy/js/core/misc.js:105]: Check what happens if redirected while other RPCs are ongoing

**Comment by aab-odoo** [comment]
robodoo rebase-ff r+

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**